support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -455,7 +455,7 @@ export function ThreeTierLanding(): JSX.Element {
 	const tier1GenericCheckoutUrlParams = new URLSearchParams({
 		product: 'Contribution',
 		ratePlan: selectedContributionRatePlan,
-		price: recurringAmount.toString(),
+		contribution: recurringAmount.toString(),
```

Comment:
Does the equivalent `price` on Tier2 Params need to be replaced with a zero `contribution`?

Reply:
`undefined` should mean there is no contribution. We shouldn't have to send a price across to `SupportPlus` as that value is derived from the product API.